### ISSUE TITLE
youtube.com: ads

### ISSUE THREAD

**Issue by chickenwithakeyboard** [uBlockOrigin/uAssets#16848]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.youtube.com/`

### Category

ads

### Description

Though the ads are blocked on the Youtube homepage, the space where the ad would be is left blank. I tried my best to make sure this is not a duplicate issue for my browser or uBlock version, but if it is, I'm very sorry.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>
![image](https://user-images.githubusercontent.com/68468595/219973586-bedca049-7195-4267-bbba-e63994a6da98.png)

### Configuration

<details>

```yaml
uBlock Origin: 1.46.0
Firefox: 109
filterset (summary): 
  network: 83045
  cosmetic: 38617
  scriptlet: 16934
  html: 885
listset (total-discarded, last-updated): 
  default: 
    user-filters: 2-0, never
    ublock-filters: 32911-55, 1d.18h.5m
    ublock-badware: 4357-5, 1d.18h.4m
    ublock-privacy: 321-0, 22h.16m
    ublock-abuse: 79-0, 22h.15m
    ublock-unbreak: 1911-0, 1d.18h.6m
    ublock-quick-fixes: 173-0, 5h.40m
    easylist: 61799-640, 4d.8m
    easyprivacy: 30963-1743, 4d.7m
    urlhaus-1: 5908-0, 1h.39m
    plowe-0: 3676-2, 9d.2h.54m
filterset (user): [array of 2 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 369 ms (selfie)
  maxAssetCacheWait: 199 ms
popupPanel: 
  blocked: 7
  blockedDetails: 
    youtube.com: 3
    doubleclick.net: 2
    google.ca: 1
    google.com: 1
```
</details>

**Comment by garry-ut99** [comment]
>  * [x]  I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)

Duplicate of https://github.com/uBlockOrigin/uAssets/issues/7636
